includes/class-wc-calypso-bridge-plugins.php

```diff
@@ -44,6 +44,7 @@ class WC_Calypso_Bridge_Plugins {
 		'woocommerce-shipment-tracking/woocommerce-shipment-tracking.php',
 		'crowdsignal-forms/crowdsignal-forms.php',
 		'polldaddy/polldaddy.php',
+		'woocommerce-product-recommendations/woocommerce-product-recommendations.php',
```

Comment:
That probably will conflict with https://github.com/Automattic/wc-calypso-bridge/pull/912/files -- Not a blocker; just pointing that out.

Reply:
Eagle eye :) I did this on purpose, so I have all the related plugins, as mentioned in the project thread, in one PR.